plugins/paper/src/main/java/com/playmonumenta/plugins/custominventories/ClassSelectionCustomInventory.java

```diff
@@ -588,6 +626,30 @@ public void addSpecItem(PlayerClass playerClass, PlayerSpec spec, PlayerSpec oth
 		}
 	}
 
+	public ItemStack createLevelItem(PlayerClass theClass, AbilityInfo<?> ability, int level, Player player, boolean isSpec) {
```

Comment:
This is only used in this gui, no need to override and have 2 versions do the same actions, just use the new parameter to define new work within the same function